xml/admin_saltcluster.xml

```diff
@@ -1300,7 +1300,19 @@ devices                   /dev/vdb
         </step>
         <step>
           <para>
-            Migrate the DB device:
+            If the OSD does not have a pre-existing DB device, create new DB with the <command>new-db</command> command:
```

Comment:
```suggestion
            If the OSD does not have a preexisting DB device, create a new DB with the <command>new-db</command> command:
```

Reply:
Huh, that's so weird it reverted - sorry @dariavladykina fixing again